.github/actions/create-release-archive/windows/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Create release archive
+description: Creates a tar archive for a release binary
```

Comment:
This one does create a `zip` archive ;)

Reply:
Correct, that should be changed to avoid future confusion.

Reply:
fixed